pandas/core/generic.py

```diff
@@ -3883,10 +3883,13 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
         limit : int, default None.
             Maximum number of consecutive NaNs to fill. Must be greater than 0.
         limit_direction : {'forward', 'backward', 'both'}, default 'forward'
-            If limit is specified, consecutive NaNs will be filled in this
-            direction.
-
+            Consecutive NaNs will be filled in this direction.
             .. versionadded:: 0.17.0
+        limit_area : {'inside', 'outside'}, default None
+            * 'inside' Only fill NaNs surrounded by valid values (interpolate).
+            * 'outside' Only fill NaNs outside valid values (extrapolate).
+            * None: default fill inside and outside
+            .. versionadded:: 0.21.0
```

Comment:
@jreback I also noticed and corrected the old .. versionadded tag on 3887 which was not being property replaced.  It needed the blank lines to stop it from being combined with the normal paragraph above.